### PR TITLE
feat: add enmasse resources to showcase deployment

### PR DESCRIPTION
**Pull request by darahayes**
What this PR does:

* Adds the necessary resources needed to set up the AMQ Online / enmasse integration in the showcase server in `datasync-showcase.yml`
* makes a copy of the original `datasync-showcase.yml` called `datasync-showcase-no-amq.yml` file that can be referenced for upstream docs/examples without the need of AMQ Online.

### CODE REVIEW

File: openshift/datasync-showcase-no-amq.yml [hunk 1]
```diff
@@ -0,0 +1,359 @@
+---
```
Comment (wtrocki) [inline openshift/datasync-showcase-no-amq.yml]: It is strange pattern. I would expect showcase and showcase-amq. 
Any reason why we do this?

Comment (darahayes) [reply openshift/datasync-showcase-no-amq.yml]: Ahh.. The main reason is just so no changes are needed in the integreatly installer. I am happy to change it though!

Comment (wtrocki) [reply openshift/datasync-showcase-no-amq.yml]: Ah.. But then would other showcase have subscription working without AMQ - probably will need some non AMQ broker to be complete anyway so we practically getting generic showcase and integrately one

**Comment by darahayes** [comment]
@wtrocki I've updated this PR. We now have two templates.

* datasync-showcase - deploys everything including AMQ stuff. Used by integreatly installer. Cannot 100% verify this right now due to issues with the integreatly installer, however I did have those AMQ sepcific resources verified before.
* datasync-showcase-community deploys a non production ready instance of mosquitto MQTT broker. Spent ages making sure this one works and I've been able to get subscriptions working with this also

**Comment by wtrocki** [comment]
I have tested showcase-commnity on community cluster. Really good work on this one. 
Let's make release and follow up with integrately installer and test the other one on RHMI but I think we can simply merge that now and do another release.

**Review by wtrocki — APPROVED** [review]

**Comment by wtrocki** [comment]
We might require documentation updates (readme) and in mobile-docs that actually have copy paste of the previous version of template (😞)

**Pull request merged**